ui/shared/utils/constants.js

```diff
@@ -1307,6 +1307,27 @@ export const PANEL_APP_CONFIDENCE_LEVEL_COLORS = Object.entries(PANEL_APP_CONFID
   (acc, [confidence, color]) => ({ ...acc, [confidence]: VARIANT_ICON_COLORS[color] }), {},
 )
 
+export const PANEL_APP_MOI_OPTIONS = [{
```

Comment:
I just meant we should define a constant somewhere like `const AR_MOI = 'AR'` and then use `AR_MOI` instead of the hardcoded string `AR` both here and  in `moiToMoiInitials` . It shouldn't require rewriting the function

Reply:
Done 👍